docs/operate/validators/docker-quickstart.mdx

```diff
@@ -0,0 +1,259 @@
+```bash
+mkdir -p ethereum/hyperlane_db && \
+touch ethereum/config.json ethereum/docker-compose.yml ethereum/.env
+```
+
+### Edit config.json
+
+`customRpcUrls` are a comma separated list of performant RPC endpoints for the chain you wish to support, we recommend using paid providers to avoid rate limiting
+
+`signer.region` and `validator.region` should be adjusted to your AWS region
+
+`signer.id` and `validator.id` should be adjusted to your AWS KMS id you configured in step 3, prefixed with `alias/`
+
+`originchainname` should reflect the chain 
+
+`checkpointsyncer.bucket` should reflect the name of the S3 bucket
+
+`checkpointsyncer.folder` is the name of the folder the validator will use within the bucket (you may use the same bucket for multiple validators, but ensure each folder name is unique per validator)
+
+`reorgperiod` may be different for each chain [**find the reorgperiods**](https://github.com/hyperlane-xyz/hyperlane-registry/tree/main/chains)
+
+```json
+{
+  "chains": {
+    "ethereum": {
+      "customRpcUrls": "https://eth.llamarpc.com,https://rpc.mevblocker.io",
+      "signer": {
+        "region": "us-east-1",
+        "type": "aws",
+        "id": "alias/hyperlane-validator-signer-ethereum"
+      }
+    }
+  },
+  "originchainname": "ethereum",
+  "db": "/mnt/hyperlane_db",
+  "validator": {
+    "id": "alias/hyperlane-validator-signer-ethereum",
+    "type": "aws",
+    "region": "us-east-1"
+  },
+  "checkpointsyncer": {
+    "bucket": "hyperlane-validator-signatures-ethereum",
+    "region": "us-east-1",
+    "type": "s3",
+    "folder": "ethereum"
+  },
+  "reorgperiod": 14,
+  "metrics": "9090"
+}
+```
+
+### Edit .env
+
+```
+AWS_ACCESS_KEY_ID=<Your AWS access key ID>
+AWS_SECRET_ACCESS_KEY=<Your AWS secret access key>
+DB_PATH=./hyperlane_db # ./hyperlane_db
+METRICS_PORT=9091 # Default is 9090, should match port set in config.json
+CONFIG_FILE=./config.json # ./config.json
+SERVICE_NAME=ethereum
+```
+
+### Edit docker-compose.yml
+
+If you wish to run multiple validators, you could reuse the same docker-compose file but create different .env and config.json files per chain. 
+
+```yaml
+version: '3.8' 
+
+services:
+  validator:
+    image: gcr.io/abacus-labs-dev/hyperlane-agent:f68319f-20240625-163648
+    command: ./validator
+    container_name: ${SERVICE_NAME-validator}
+    ports:
+      - "${METRICS_PORT}:9090/tcp"
+    environment:
+      CONFIG_FILES: /config.json
+      AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
+      AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
+    configs:
+      - config.json
+    volumes:
+      - ${DB_PATH}:/mnt/hyperlane_db
+    restart: unless-stopped  # Ensure the container restarts unless explicitly stopped
+    env_file:
+      - .env
+
+configs:
+  config.json:
+    file: ${CONFIG_FILE}
+```
+
+## Step 6. Run the Hyperlane validator 
+
+Bring the container up 
+
+```bash
+cd ethereum && docker-compose up -d
+```
```

Comment:
As mentioned above, would be good to rework the config + these steps to run, so that there is no need to cd into individual dirs

Reply:
also worth adding info of how to run subset of validators e.g. `docker-compose up -d ethereum` to just run ethereum validator